static/src/javascripts/bootstraps/video-embed.js

```diff
@@ -16,7 +15,7 @@ define([
     'common/modules/video/fullscreener',
     'common/views/svgs',
     'text!common/views/ui/loading.html',
```

Comment:
I didn't because it's just a string (there are a few other examples), but if it makes things clearer then I'm doing it.